### ISSUE TITLE
clearing invalidation should send them to "acknowledged"

### ISSUE THREAD

**Issue by vlimant** [cms-PdmV/cmsPdmV#1163]
looks like a button or an api is not doing what it should.
the "clear" for invalidation should send it to `acknowledged` instead of `announced` (to be operated by ops).

the api says "Clear selected invalidations without announcing" https://cms-pdmv-prod.web.cern.ch/mcm/restapi/invalidations 

so that's what it should do.

**Comment by vlimant** [comment]
looking again, to me "clearing" means "forget about it" == "delete" (which exists. the current behaviour is to clear it into the announced list, which is the same a announcing.

I vote to  remove the clear restapi and button

**Comment by ggonzr** [comment]
The endpoint used to serve its purpose. Looking at the history via:

```bash
git log --reverse -L '/class Clearer/,/^class GetInvalidation'/:rest_api/InvalidationActions.py
```
The commit 52fb5d524dfba03b063778aa25fffeb603761e6c includes that feature, `/clear` could have meant: "set the invalidation to the status `announced` without performing a real invalidation (asking the data operations team via email to invalidate the linked dataset in DBS)". This is not currently useful as my understanding is this process currently managed by CMS Unified is performed automatically by querying the McM database scanning invalidation objects in the `announced` status and then performs the respective invalidation. In conclusion, the only useful point this is doing now is not sending the email.

I do not think it is a good idea to use this endpoint to send these objects to `acknowledged` as they could create possible inconsistencies between DBS, Unified and McM. I suggest to just remove the `/clear` endpoint as now it is dead code.

**Comment by vlimant** [comment]
it might even be me who wrote that commit :-(

**Comment by ggonzr** [comment]
> at the time, indeed "announced" meant "operated" for mcm user ; as this would mean that the email was send and should have been operated on.
> 
> is delete available to requests managers ?

No, just for administrators ([Details](https://github.com/cms-PdmV/cmsPdmV/blob/b31fb59f33ec57bf32cbcc63591ff290592df6c7/mcm/rest_api/InvalidationActions.py#L88-L91)). Also, I am not sure if it is a good idea to allow them doing this as this could be prone to create inconsistencies (Invalidation is triggered after a submitted/done request is reset, if by mistake the record is deleted, an inconsistency is created). So I think the plan related to just delete the endpoint is the most suitable approach in my opinion.

**Comment by vlimant** [comment]
leaving the hold (request manager) -> delete (admin only) is indeed an option as the use case of needing to leave a dataset valid even after the original request was reset is quite rare (but not zero)
